app/views/courses/index.html.erb

```diff
@@ -1,3 +1,21 @@
-<p>Display all courses for current user.</p>
+<% unless Course.all.empty? %>
+<p>You are currently associated with the following courses.</p>
 
-<%= current_user.email %>
\ No newline at end of file
+<% @courses.each do |course| %>
```

Comment:
This can be easily replaced with 

    <%= render @courses %>

Then define a partial called `_course.html.erb` in the courses folder that has the panel for each view.

I will show you during the OSS meeting.